pkg/internal/runtime/runtime.go

```diff
@@ -18,6 +18,8 @@ func GetDefault(logger log.Logger) cluster.ProviderOption {
 	case "docker":
 		logger.Warn("using docker due to KIND_EXPERIMENTAL_PROVIDER")
 		return cluster.ProviderWithDocker()
+	case "nerdctl", "finch":
```

Comment:
Could you add a CI?
ref:
- https://github.com/kubernetes-sigs/kind/pull/3408

Reply:
yeah, I saw that PR.. should definitely collaborate on getting this into CI. Wanted to make sure there weren't any glaring misses here before getting CI set up.

Reply:
> what is finch?

Amazon's distribution of nerdctl wrapped in Lima
https://aws.amazon.com/jp/blogs/opensource/ready-for-flight-announcing-finch-1-0-ga/

Reply:
Are there compatibility issues such that we need to expand the test matrix that far as both upstream nerdctl and the finch distro?